ios/PushNotificationPlugin/PushNotificationPlugin.m

```diff
@@ -610,4 +619,41 @@ - (void)failIfSimulator {
     }
 }
 
+- (void)getTagsFromServer:(void (^)(NSArray *tags))handler {
```

Comment:
Could you add a comment that explains why we decided to implement the tags feature the way we did? That will probably be very helpful to future generations :)